packages/content/src/collections/squiggleAiLibraries.ts

```diff
@@ -0,0 +1,60 @@
+import { defineCollection } from "@content-collections/core";
+
+const GRAPHQL_URL = "https://squigglehub.org/api/graphql";
+
+export function getQuery(owner: string, slug: string) {
+  return `
+  query GetModelCode {
+    model(input: {owner: "${owner}", slug: "${slug}"}) {
+      ... on Model {
+        id
+        currentRevision {
+          content {
+            ... on SquiggleSnippet {
+              id
+              code
+            }
+          }
+        }
+      }
+    }
+  }
+  `;
+}
+
+export async function fetchCodeFromGraphQL(
+  owner: string,
+  slug: string
+): Promise<string> {
+  const query = getQuery(owner, slug);
+  const response = await fetch(GRAPHQL_URL, {
+    headers: {
+      "Content-Type": "application/json",
+    },
+    method: "POST",
+    body: JSON.stringify({ query }),
+  });
+  const json = await response.json();
+  const code = (json as any).data.model.currentRevision.content.code;
+  if (typeof code !== "string") {
+    throw new Error(`Invalid response: ${JSON.stringify(json, null, 2)}`);
+  }
+  return code;
+}
```

Comment:
Copy-pasted from AI where similar code is used for fine-tuning.

In the future, we can move both to `packages/hub-client` library.